chore(unittest): write unittests for all endpoints

danwebguy opened the pull request:
### What does this PR do?
Test all Enpoints

### Description of Task to be completed?
- unit test for meetups
- unit test for questions
- unit test for rsvps, comments
- bit of refactoring

### How should this be manually tested?
npm test

### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/n/projects/2232401

deliver #163588063

danwebguy committed:
chore(unittest): write unittests for all endpoints

- unit test for meetups
- unit test for questions
- unit test for rsvps, comments
- bit of refactoring

finishes #163588063

Orlayhemmy commented on server/controllers/questions.js:
```diff
@@ -33,8 +33,8 @@ class questionsController {
       const results = await db.query(findAllQuestions);
-      return res.status(200).json({
-        status: 200,
+      return res.status(201).json({
```
When you make a query, the status code is 200. You only use 201 for post

Orlayhemmy commented on server/test/comments.js:
```diff
@@ -0,0 +1,24 @@
+chai.use(chaiHttp);
+chai.should();
+
+const token = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyaWQiOjIsImFkbWluIjp0cnVlLCJpYXQiOjE1NDgyMjIzMDJ9.jeI-rw4tS4lbj7m1PzXLf0i75D6CwRTmmq_ExiArOhY';
```
why don't you store the token in your env?

Orlayhemmy commented on server/test/comments.js:
```diff
@@ -0,0 +1,24 @@
+
+const token = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyaWQiOjIsImFkbWluIjp0cnVlLCJpYXQiOjE1NDgyMjIzMDJ9.jeI-rw4tS4lbj7m1PzXLf0i75D6CwRTmmq_ExiArOhY';
+
+describe('/Post comments/:questionid', () => {
```
Instead of multiple Describes, use one describe and multiple ITs. A file is expected to have one Describe block

Orlayhemmy reviewed (commented):
A chore doesn't use finishes in the commit message footer, it uses deliver instead.

danwebguy committed:
chore(unittest): write unittests for all endpoints

- unit test for meetups
- unit test for questions
- unit test for rsvps, comments
- bit of refactoring

deliver #163588063
